src/components/ApplicationGroup/List/EnvironmentsList.tsx

```diff
@@ -11,7 +11,7 @@ import { Cluster } from '../../../services/service.types'
 import { AppListConstants } from '../../../config'
 import { useHistory, useLocation } from 'react-router-dom'
 
-export default function EnvironmentsList() {
+export default function EnvironmentsList({isSuperAdmin}:{isSuperAdmin: boolean}) {
```

Comment:
Please move this to type also format the file.

Reply:
done